global.json

```diff
@@ -3,13 +3,13 @@
     "allowPrerelease": true
   },
   "tools": {
-    "dotnet": "8.0.201",
+    "dotnet": "9.0.100-preview.6.24328.19",
     "vs": {
       "version": "17.8.0"
     },
     "xcopy-msbuild": "17.8.5"
   },
   "msbuild-sdks": {
-    "Microsoft.DotNet.Arcade.Sdk": "8.0.0-beta.24376.1"
+    "Microsoft.DotNet.Arcade.Sdk": "9.0.0-beta.24401.1"
```

Comment:
To be decided in scope of the: https://github.com/dotnet/dnceng/issues/3713

Reply:
Resolving the comment. 
Current situation is following: 
Currently there is no blockers for merging current PR in terms of versions of the Arcade. 
In case there will be any servicing request for vs17.12 at a time when the dotnet 9 will be retired, the arcade will need to be upgraded.